libs2ecore/include/s2e/CorePlugin.h

```diff
@@ -373,6 +373,7 @@ class CorePlugin : public Plugin {
     ///
     sigc::signal<void,
                  S2EExecutionState*,
+                 const klee::ref<klee::Expr>* /*condition*/,
```

Comment:
It's not clear if you want plugins to modify the condition. Probably not, given the const. In this case, use & instead of *. Users won't have to use * to dereference it, it's more convenient.

Reply:
On that note, *bool for allow forking should probably also be a reference instead of a pointer.

Reply:
Thanks! I've changed * to & for both parameters: `condition` and `allowForking`.